content/en/docs/concepts/configuration/secret.md

```diff
@@ -1,16 +1,23 @@
 ---
 reviewers:
 - mikedanese
-title: Secrets
+title: 
```

Comment:
Why did you remove the title? Now https://deploy-preview-8995--kubernetes-io-master-staging.netlify.com/docs/concepts/configuration/secret/ does not show a page title (only "Kubernetes" - not as previously "Kubernetes - Secrets")

Reply:
That was inadvertent .... let me put it back.

Reply:
Fixed.